TimberAPI/TimberAPI.cs

```diff
@@ -31,10 +30,5 @@ public static class TimberAPI
         /// APIs to load and fetch assets
         /// </summary>
         public static IAssetRegistry AssetRegistry = new AssetRegistry();
-
-        /// <summary>
-        /// APIs to add Objects to be loaded by the game
-        /// </summary>
-        public static ICustomObjectRegistry CustomObjectRegistry = new CustomObjectRegistry();
```

Comment:
Does this mean we no longer add buildings from code? If so, then an example on how to add buildings with the new system would be needed.

This change also breaks the CystomObjectRegistryExample? The example should then be removed or fixed